fem/fespace.hpp

```diff
@@ -388,10 +486,21 @@ class FiniteElementSpace
    /// Returns vector dimension.
    inline int GetVDim() const { return vdim; }
 
-   /// Returns the order of the i'th finite element
-   int GetOrder(int i) const;
-   /// Returns the order of the i'th face finite element
-   int GetFaceOrder(int i) const;
+   /// Returns the polynomial degree of the i'th element.
+   /** This is normally the same as GetElementOrder(i), except for RT finite
+       element spaces, where the actual degree of the RTp collection is (p+1).*/
+   int GetElementDegree(int i) const { return GetFE(i)->GetOrder(); }
+
+   /// Returns the polynomial degree of the i'th finite element.
+   /** Deprecated: please use @a GetElementDegree instead. */
+   MFEM_DEPRECATED int GetOrder(int i) const { return GetElementDegree(i); }
+
+   /** Returns the order of the specified edge. In a variable order space, order
+       of the given variant is returned, or -1 if there are no more variants. */
+   int GetEdgeOrder(int edge, int variant = 0) const;
```

Comment:
Some additional comments on what `variant` means would be appreciated.

Reply:
Added.